src/ocean/core/SmartUnion.d

```diff
@@ -325,7 +325,7 @@ unittest
     TestSmartUnion u;
     u.a = 1;
 
-    with (TestSmartUnion.Active) final switch (u.active)
+    with (TestSmartUnion.Active) /* final */ switch (u.active)
```

Comment:
Could you create an issue describing when we should reenable this?

Reply:
Done

Reply:
I would comment the whole unittest, since the point is to test `handleInvalidCases`, isn't it ?

Reply:
Yeah, I would just comment entire unittest.